### DISCUSSION TITLE
What are “bill subjects”

### DISCUSSION THREAD

**Discussion by jtheletter** [openstates/issues#575] [Q&A] [answered]
I’ve noticed that bill pages include (below the Follow button, above the Bill Sponsors) a section titled “Bill Subjects”, below which includes a short list of buttons that don’t do anything. I don’t see these mentioned in the OpenStates blog or Q&A.

Would this simply be a feature that was created with intention to expand later, or is this something developed that I’m missing? Thank you.

**Comment by jamesturk** [comment, marked as answer]
Bill Subjects are added by the state & just represented on our site as-is, no planned feature beyond that. 

The fact that they look like buttons is unintentional, they're basically tags, sorry for the confusion!

**Comment by jtheletter** [comment]
Thanks for your quick reply. 

The reason I thought they were buttons is because they respond to mouseover with a change of background color and change of mouse pointer (like other buttons on OpenStates do). Perhaps their style should be changed?

I was hoping I might be able to subscribe to bill subjects, instead of manually searching for new bills’ introductions.

Thanks again.
